.github/workflows/CI.yml

```diff
@@ -343,29 +344,76 @@ jobs:
       # Build and archive image into cache
       - uses: actions/checkout@v2
 
-      - name: Cache Image
-        id: cached_image
-        uses: actions/cache@v2
-        with:
-          key: "cached_image_${{ github.run_id }}"
-          path: "${{ env.PACKAGING_ROOT }}"
-
-      - name: restore cached binaries
+      - name: Restore cached binaries
         if: steps.cached_image.outputs.cache-hit != 'true'
         uses: actions/cache@v2
         with:
           key: "cached_binaries_${{ github.run_id }}"
           path: build
 
-      - name: Build Image
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to Public ECR
+        if: steps.cached_image.outputs.cache-hit != 'true'
+        id: login-ecr
+        uses: docker/login-action@v1
+        with:
+          registry: public.ecr.aws
+          username: ${{ secrets.INTEG_TEST_AWS_KEY_ID }}
+          password: ${{ secrets.INTEG_TEST_AWS_KEY_SECRET }}
+        env:
+          AWS_REGION: us-east-1
+
+      - name: Versioning for adot collector image
+        id: versioning
+        run: |
+          Version="`cat VERSION`-$GITHUB_RUN_ID"
```

Comment:
If it's same behavior as before can be in another PR but shouldn't this use commit ID instead of run ID?

Reply:
Also good if we can use the metadata action

https://github.com/docker/metadata-action

Reply:
True thought. If it is the same behavior, using Commit ID would be better than Run ID.

Reply:
Where is `$GITHUB_RUN_ID` coming from? Shouldn't it be `${{ github.run_id }}`. I agree that a short commit SHA would be more useful since no one knows the run IDs, but that change should be in another PR.

Reply:
Its the [same so](https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables) should be fine. Agree to be in another PR.